includes/helpers.php

```diff
@@ -5,10 +5,10 @@
 /**
  * Get all the checklist items that have values stored.
  *
- * @return array
+ * @return string|array
  */
-function get_wp_launch_checklist_values() : array {
-	return get_option( WP_LAUNCH_CHECKLIST_SLUG . '_values', array() );
+function get_wp_launch_checklist_values() : string|array {
```

Comment:
@damonsharp tried it out. It does give a fatal error on php 7.4 due to the | in this line. But in PHP 8 works just fine.